app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -238,6 +238,7 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
             // FIXME: Need [WebRenderFragment] as lifeCycle owner until NavOverlayFragment breakout
             pinnedTileViewModel = this@WebRenderFragment.pinnedTileViewModel
             lifeCycleOwner = this@WebRenderFragment.viewLifecycleOwner
+            observeForMegaTile(this@WebRenderFragment)
             initTiles()
```

Comment:
Nit: can we rename this to `initPinnedTiles`? It's a little confusing because we're observing on the Megatile before the init for the tiles, but in reality they're not related tiles.

Reply:
Good call.  It's safe to move it one line higher too, which makes a lot more sense.  I should have done that in the first place.